examples/simple_scene.py

```diff
@@ -10,21 +10,27 @@
 
 scene = vv.Scene()
 
-t1 = vv.Mesh(vv.Geometry(), vv.TriangleMaterial())
+
+class Triangle(vv.WorldObject):
+    def __init__(self, material):
+        super().__init__()
+        self.material = material
+
+
+t1 = Triangle(vv.TriangleMaterial())
 scene.add(t1)
 
-for i in range(20):
-    scene.add(vv.Mesh(vv.Geometry(), vv.TriangleMaterial()))
+for i in range(2):
+    scene.add(Triangle(vv.TriangleMaterial()))
 
 
-camera = vv.PerspectiveCamera(45, 16 / 9, 0.1, 1000)
+camera = vv.NDCCamera()  # This material does not use a camera
 
 
 def animate():
     width, height, ratio = canvas.get_size_and_pixel_ratio()
-    camera.aspect = width / height
-    camera.update_projection_matrix()
-    # Actually render the scene
+    camera.update_viewport_size(width, height)
```

Comment:
why doesnt this need `camera.update_projection_matrix()`?

Reply:
The renderer calls `update_projection_matrix()`, I guess the other examples update it twice :)

Reply:
Sry, only the `scene_in_qt_subwidget.py` still did it in the example.

Reply:
> The renderer calls `update_projection_matrix()`, I guess the other examples update it twice :)

I realize that it did do that at some point, but it doesn't anymore. The user is required to do it because they are the ones updating the camera, so that's better than calling it every frame...